core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -1223,6 +1234,7 @@ public ClientDependencyInfo getClientDependencyInfo(int dependencyId)
   public ClientFileInfo getClientFileInfo(int fid) {
     synchronized (mRootLock) {
       Inode inode = mFileIdToInodes.get(fid);
+      mMasterSource.incGetFileStatusOps();
```

Comment:
I think that is the more logical way of measuring `*Ops`. It will give the user insight into which code paths are being heavily used (or not called), and successes are mostly covered by the `num*` metrics.